Lab Member YAMLs/Templates/Full Name with Spaces.yaml

```diff
@@ -0,0 +1,46 @@
+name:
+birthday: MM.DD
+position: (Job Title) (Graduate Student) (Postoctoral Research Scholar/Fellow)
+affiliations: >
+  Weill Cornell Physiology, Biophysics, and Systems Biology PhD Program
+  Weill Cornell Biochemistry and Structural Biology PhD Program
+  Tri-Institutional PhD Program in Computational Biology and Medicine
+  Tri-Institutional PhD Program in Chemical Biology
+  Louis V. Gerstner, Jr. Graduate School of Biomedical Sciences
+  Berlin Institute of Health (BIH) Einstein Foundation - Stiftung Charité Group
+  Joint Mentorship with Name (URL to lab)
+fellowships: >
+ (Any awards you have won) (i.e. MolSSI Investment Fellowship)
+funding: >
+ (list of funding sources)
+status: current
+  start-date: YYYY.MM.DD
+  end-date: YYYY.MM.DD
+email: @choderalab.org
+telephone:
+twitter: handle (NO@)
+linkedin: URL
+ORCID:
+google-scholar: URL
+CV: (URL to PDF)
+project(s): >
```

Comment:
I think `project(s):` is invalid.
Let's do
```yaml
projects:
  - espaloma
  - perses
  - ml-potential
```
Somewhere else, we'd need to capture a list of projects with the same keywords.

Reply:
Alternatively, we can solicit text descriptions of the projects, which is the more useful thing to do right now!
```yaml
# Brief descriptions of each project you are involved in (to appear on webpage)
projects:
- End-to-end differentiable fitting of molecular mechanics force fields
- Alchemical free energy calculations that can learn
- Machine learning for drug discovery
- Large-scale molecular dynamics simulations of transmembrane proteins
```